.claude/skills/managing-translations/SKILL.md

```
@@ -45,24 +45,24 @@ python3 scripts/sync_data.py --templates-dir templates
+`scripts/data/i18n.json` stores:
 
 - Translation status for each template (per language).
 - Tag translation mappings (English → each language).
 - Category title translations.
 
 ## Adding Translations for a Template
 
-1. Edit `scripts/i18n.json` to add translations under the template's name.
+1. Edit `scripts/data/i18n.json` to add translations under the template's name.
 2. Run the sync script:
    ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)